raiden-dapp/src/service-worker/assistant.ts

```diff
@@ -1,15 +1,34 @@
 /* istanbul ignore file */
 import { Store } from 'vuex';
 import compareVersions from 'compare-versions';
+import { ServiceWorkerMessages, ServiceWorkerAssistantMessages } from './messages';
 import { CombinedStoreState } from '@/store/index';
 
 const VERSION_FILE_PATH = (process.env.BASE_URL ?? '/') + 'version.json';
 
 export default class ServiceWorkerAssistant {
   constructor(private store: Store<CombinedStoreState>) {
+    navigator.serviceWorker.onmessage = this.onMessage;
     setInterval(this.updateAvailableVersion, 1000 * 60);
   }
 
+  public update = (): void => {
+    navigator.serviceWorker.controller?.postMessage(ServiceWorkerAssistantMessages.UPDATE);
+  };
+
+  private onMessage = (event: MessageEvent): void => {
+    if (!event.data) return;
+
+    switch (event.data) {
```

Comment:
Out of curiosity, why did you decide to go with a `switch` statement here when you are not really switching between things? Or maybe you are adding more stuff in a later commit?

Reply:
Because I know the future. :wink: 
Also was this the code approach before this PR and the MDN suggest this approach. Also all these arguments are really weak. In first place because I knew there are more messages to come.